repo/packages/C/crate/0/config.json

```diff
@@ -0,0 +1,156 @@
+          "description": "The absolute path where Crate should store table data. Note that the user `crate` needs write access!"
+        },
+        "blob-path": {
+          "type": "string",
+          "description": "The absolute path where Crate should store blobs. Note that the user `crate` needs write access!"
+        },
+
+        "framework": {
+          "description": "Framework configuration properties.",
+          "type": "object",
+          "properties": {
+            "zookeeper": {
+              "type": "string",
+              "description": "URL to the Zookeeper Crate will communicate to (without protocol prefix, e.g. mesos-master:2181)"
+            },
+            "cpus": {
+              "type": "number",
+              "description": "Required number of cpus.",
+              "default": 0.25,
+              "minimum": 0.25,
+              "maximum": 1
+            },
+            "instances": {
+              "type": "integer",
+              "description": "Number of framework instances.",
+              "default": 1,
+              "minimum": 1,
+              "maximum": 3
+            },
+            "memory": {
+              "type": "integer",
+              "description": "Required heap size.",
+              "default": 128,
+              "minimum": 128,
+              "maximum": 256
+            },
+            "api-port": {
+              "type": "integer",
+              "description": "Port for the REST API of the Crate Mesos Framework.",
+              "default": 4040,
+              "minimum": 0,
+              "maximum": 65535
+            },
+            "user": {
+              "type": "string",
+              "description": "User to run the framework as."
+            },
+            "role": {
+              "type": "string",
+              "description": "Mesos role for the framework."
+            },
+            "auth": {
+              "description": "Authentication Configuration Properties.",
+              "type": "object",
+              "properties": {
+                "enabled": {
+                  "type": "boolean",
+                  "description": "Whether framework authentication should be enabled. Requires principal and secret to be set.",
+                  "default": false
+                },
+                "principal": {
+                  "type": "string",
+                  "description": "The Mesos principal used for framework authentication."
+                },
+                "secret": {
+                  "type": "string",
+                  "description": "The Mesos secret used for framework authentication."
+                }
+              },
+              "required": []
+            }
+          },
+          "required": [
+            "zookeeper"
+          ]
+        },
+
+        "resources": {
+          "description": "Crate resources configuration properties.",
+          "type": "object",
+          "properties": {
+            "cpus": {
```

Comment:
cpus is defined under framework and it is defined here.  It only needs to be defined once.  This section should be deleted.